pkg/apitype/policy.go

```diff
@@ -64,12 +68,31 @@ type Policy struct {
 	DisplayName string `json:"displayName"`
 
 	// Description is used to provide more context about the purpose of the policy.
-	Description      string           `json:"description"`
+	Description string `json:"description"`
+
+	// This enforcement level will be deprecated in favor of enforcement
```

Comment:
An individual policy will always declare its enforcement level, right? It's just that the policy pack's configuration can _overwrite_ that enforcement level? I'm not sure what we expect this to be long term, as I would have expected us to keep the `EnforcementLevel` field on the `Policy` type.

Reply:
The `DefaultConfiguration` is set at the Policy level, so `EnforcementLevel` is still at the Policy level just embedded within the `DefaultConfiguration` to make it clear to authors that the value can be overriden.